.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Install 🔧
-        uses: viamrobotics/js-config/.github/actions/setup@0006127efb3f62669f662ca31c555a49a034935e
+        uses: viamrobotics/js-config/.github/actions/setup@7e4e4f1bd14c83aaec2fd877324c38028d8a387b
```

Comment:
[opt] swap this to be unpinned like app.

Reply:
My opinion is that we should err on the side of consistency.

Reply:
:+1: Changed to pin to `@main`